bundle/edu.gemini.pot/src/main/scala/edu/gemini/spModel/gemini/igrins2/Igrins2.scala

```diff
@@ -0,0 +1,227 @@
+package edu.gemini.spModel.gemini.igrins2
+
+import edu.gemini.pot.sp.{ISPNodeInitializer, ISPObsComponent, ISPObservation, SPComponentType}
+import edu.gemini.shared.util.immutable
+import edu.gemini.shared.util.immutable.{DefaultImList, ImList}
+import edu.gemini.spModel.core.{Angle, MagnitudeBand, Site, Wavelength}
+import edu.gemini.spModel.data.ISPDataObject
+import edu.gemini.spModel.data.config.{DefaultParameter, DefaultSysConfig, ISysConfig, StringParameter}
+
+import java.util.{Collections, Map => JMap, Set => JSet}
+import edu.gemini.spModel.data.property.{PropertyProvider, PropertySupport}
+import edu.gemini.spModel.gemini.init.ComponentNodeInitializer
+import edu.gemini.spModel.gemini.parallacticangle.ParallacticAngleSupportInst
+import edu.gemini.spModel.obscomp.InstConstants
+import edu.gemini.spModel.pio.{ParamSet, Pio, PioFactory}
+import edu.gemini.spModel.seqcomp.SeqConfigNames
+import edu.gemini.spModel.telescope.{IssPort, IssPortProvider, PosAngleConstraint, PosAngleConstraintAware}
+import squants.time.Time
+import squants.time.TimeConversions.TimeConversions
+
+import java.beans.PropertyDescriptor
+import scala.collection.immutable.TreeMap
+import scala.collection.JavaConverters._
+import scala.math.sqrt
+
+/*
+ ** The Igrins2 instrument SP model.
+ * Note that we do not override clone since private variables are immutable.
+ */
+final class Igrins2 extends ParallacticAngleSupportInst(Igrins2.SP_TYPE) with PropertyProvider with PosAngleConstraintAware with Igrins2Mixin with IssPortProvider {
```

Comment:
Should we specify a serial version?

Reply:
Do we ever use that? I've never changed or upgraded one before.

Reply:
Probably not.  Mostly we only change the binary format between major releases, when we do a migration anyway.  If we add one and then make a breaking change in a supposedly minor release it would only cover it up.